### PR TITLE
USHIFT-1584: Parameterize instance disk sizes

### PR DESCRIPTION
**Pull request by copejon**
EC2 instance tests require 2 disks, but smaller overall sizes, where as metal test require 1 large disk.  This PR makes the disk sizes configurable per job, and defaults to 100 and 20 for primary and secondary sizes.  The metal workflow overrides these with 300 and 0.

### CODE REVIEW

File: ci-operator/step-registry/openshift/microshift/infra/aws/ec2/openshift-microshift-infra-aws-ec2-commands.sh [hunk 1]
```diff
@@ -44,8 +26,20 @@ echo -e "AMI ID: $ami_id"
...
+Conditions:
+  AddSecondaryVolume: !Not [!Equals [!Ref EC2Type, 'MetalMachine']]
+Mappings:
+ VolumeSize:
+   MetalMachine:
+     PrimaryVolumeSize: "300"
+     SecondaryVolumeSize: "0"
+   VirtualMachine:
+     PrimaryVolumeSize: "200"
+     SecondaryVolumeSize: "100"
```
Comment (ggiguash) [inline ci-operator/step-registry/openshift/microshift/infra/aws/ec2/openshift-microshift-infra-aws-ec2-commands.sh:38]: I would rather go with 20+10 sizes here. Do we need so much disk space for e2e tests?

**Comment by copejon** [comment]
The arm tests are all expected to fail, however this one shouldn't be. Giving it a little nudge 

/pj-rehearse pull-ci-openshift-microshift-release-4.13-e2e-loadbalancer-smoke-test

**Pull request merged**